src/molecules/formfields/TextField/index.js

```diff
@@ -35,13 +37,23 @@ function TextField( props ) {
             { label }
           </label>
         }
-
-        <input
-          className={styles.input}
-          type='text'
-          placeholder={placeholder}
-          {...input}
-        />
+        { phone ?
```

Comment:
Mask seems useful, I think this should be switch out on mask and have the mask pattern and char as an available prop on TextField